.github/workflows/weekly-summaries.yml

```
@@ -38,6 +38,12 @@ jobs:
           echo "IS_MONTH_START=$(date +'%d')" >> $GITHUB_ENV
           echo "DATE_START=2024-10-15" >> $GITHUB_ENV
           echo "DATE_END=$(date +'%Y-%m-%d')" >> $GITHUB_ENV
+          # Calculate days since project start
+          start_date="2024-10-15"          
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)